releasenotes/notes/feat-time-extension-chatprompt-builder-fb159207088943d8.yaml

```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Users can now work with date and time in the ChatPromptBuilder.
+    In the same way as the PromptBuilder, the ChatPromptBuilder now supports arrow to work with datetime.
```

Comment:
@dfokina I noticed that I don’t think we have in depth docs on how to use this especially around the expected format. Could we add them to the PromptBuilder and ChatPromptBuilder?